setup.py

```diff
@@ -43,6 +43,17 @@ def run(self):
 with open('bear-requirements.txt') as requirements:
     bear_required = requirements.read().splitlines()
 
+version = sys.version_info  # Checks if python version is <3.4
+# If yes changes the vim-vint~=x.x to vim-vint=x.x
+if(version[0] == 3 and version[1] <= 4):
```

Comment:
See https://github.com/coala/coala/issues/3878#issuecomment-288269752 .
This doesnt include when the version is 2.x, which is where the problem is.